pages/project-version/middleware/index.js

```diff
@@ -10,34 +10,37 @@ const getVersion = () => (req, res, next) => {
     .catch(next);
 };
 
+const extractComments = task => {
```

Comment:
Minor beef - this is in a middleware module and is not a middleware.

Reply:
I'd create `./pages/project-version/lib/extract-comments.js` and put this function there.

Reply:
done